DESCRIPTION

```diff
@@ -43,4 +43,4 @@ Imports:
 VignetteBuilder: knitr
 URL: https://github.com/sailuh/kaiaulu
 BugReports: https://github.com/sailuh/kaiaulu/issues
-RoxygenNote: 7.0.2
+RoxygenNote: 7.2.3
```

Comment:
You should not "git add" files which are not needed for the change. This is side effect behavior! This commit should be reverted to not modify this file.